ai/default/aiunit.cpp

```diff
@@ -1879,7 +1879,7 @@ static void dai_military_attack(struct ai_type *ait, struct player *pplayer,
     };
   } else {
     /* You can still have some moves left here, but barbarians should
-       not sit helplessly, but advance towards nearest known enemy city */
+   not sit helplessly, but advance towards nearest known enemy city */
```

Comment:
Don't reindent comments. I think you get the idea, not commenting on the other ones...